docs/en/observability/profiling-advanced-configuration.asciidoc

```diff
@@ -7,8 +7,6 @@ See the following sections for more information:
 * <<profiling-tag-data-query, Tag data for querying>>: Tag data collected by the host-agent into multiple logical groups so they can be queried in Kibana.
 * <<profiling-add-symbols, Add symbols for native frames>>: Push symbols to your cluster so you can see function names and line numbers in traces of applications written in programming languages that compile to native code (C, C++, Rust, Go, etc.).
 * <<profiling-use-a-proxy,Use a proxy>>:  Set up an HTTP proxy if your infrastructure host-agent installation needs one to reach {ecloud}.
-* <<profiling-probabilistic-profiling, Configure probabilistic profiling>>: Configure the Universal Profiling agent to run in probabilistic profiling mode.

```

Comment:
Is this a random fix?

Reply:
No. I've moved this intentionally because, related to ILM, probabilistic profiling can be used to control how much storage profiling requires.